pom.xml

```diff
@@ -30,6 +31,18 @@
       <artifactId>jetty-proxy</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+      <version>${junit.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock</artifactId>
+      <version>1.58</version>
```

Comment:
I go back and forth.  If it’s version used by a number of jar files that are related, then I like using the interpolation.  If it’s a single jar, then it’s yet another level of indirection.

Regardless, good to see…
> On May 18, 2016, at 10:59 AM, Matt Overstreet <notifications@github.com> wrote:
> 
> In pom.xml <https://github.com/o19s/grand_central/pull/9#discussion_r63719588>:
> 
> > @@ -30,6 +31,18 @@
> >        <artifactId>jetty-proxy</artifactId>
> >        <version>${jetty.version}</version>
> >      </dependency>
> > +    <dependency>
> > +      <groupId>junit</groupId>
> > +      <artifactId>junit</artifactId>
> > +      <scope>test</scope>
> > +      <version>${junit.version}</version>
> > +    </dependency>
> > +    <dependency>
> > +      <groupId>com.github.tomakehurst</groupId>
> > +      <artifactId>wiremock</artifactId>
> > +      <version>1.58</version>
> no reason, I'll fix that.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub <https://github.com/o19s/grand_central/pull/9/files/36fa649c8da1d0d74db04350b6894abcc4028e73#r63719588>

-----------------------------------------------------
Eric Pugh | Principal | OpenSource Connections, LLC | 434.466.1467 | http://www.opensourceconnections.com <http://www.opensourceconnections.com/> | My Free/Busy <http://tinyurl.com/eric-cal>  
Co-Author: Apache Solr Enterprise Search Server, 3rd Ed <https://www.packtpub.com/big-data-and-business-intelligence/apache-solr-enterprise-search-server-third-edition-raw>	
This e-mail and all contents, including attachments, is considered to be Company Confidential unless explicitly stated otherwise, regardless of whether attachments are marked as such.